modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/secret/management/v1/SecretManagementTestBase.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.rest.api.server.secret.management.v1;
+
+import io.restassured.RestAssured;
+import org.apache.commons.lang.StringUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.wso2.identity.integration.test.rest.api.server.common.RESTAPIServerTestBase;
+
+import java.io.IOException;
+
+/**
+ * Base test class for the Secret Management Rest APIs.
+ */
+public class SecretManagementTestBase extends RESTAPIServerTestBase {
+
+    public static final String API_DEFINITION_NAME = "secretManager.yaml";
+    public static final String API_VERSION = "v1";
+    public static final String API_PACKAGE_NAME = "org.wso2.carbon.identity.api.server.secret.management.v1";
+    public static final String SECRET_TYPE_API_BASE_PATH = "/secret-type";
+    public static final String SECRET_API_BASE_PATH = "/secrets";
+    public static final String PATH_SEPARATOR = "/";
+    public static final String SECRET_TYPE = "ADAPTIVE_AUTH_CALL_CHOREO";
+
```

Comment:
Unnecessary newline

Reply:
Addressed with commit [252de16](https://github.com/wso2/product-is/pull/12546/commits/252de169987046a367e9c09f983731935058c944)